packages/next/server/web/spec-extension/fetch-event.ts

```diff
@@ -10,13 +10,36 @@ export class NextFetchEvent extends FetchEvent {
     this.sourcePage = params.page
   }
 
-  // @ts-ignore
+  /**
+   * @deprecated The `request` is now the first parameter and the API is now async
+   * with the form:
+   *
+   * ```ts
+   * export function middleware(request, event) {
+   *   return new Response("Hello " + request.url)
+   * }
+   * ```
+   *
```

Comment:
```suggestion
```
Might be good to keep this more minimal as they can see more verbose examples in the link below.

Reply:
@ijjk I don't know about other code editors, but VS Code is very good at this, it renders the code sample very nicely and if it's large it adds a scroll.

Reply:
Yeah was more thinking it would be easier to ensure this gets updated in the error doc than in JSDoc comments if we ever change the signature or anything.